blocks/double-chain-block/chains/double-chain/default.scss

```diff
@@ -1,34 +0,0 @@
-.double-chain {
-  .small-promo {
```

Comment:
eww this is nasty before. good catch

Reply:
@matthewroach is this styles added elsewhere or is it invalid?

Reply:
Not needed, it was actually causing the need for additional CSS to overcome itself